mstats: test_kurtosis fails (ULP max is 2)

Hi, today I came across and wanted to report this minor issue in a cluster environment, using Python 3.4.3 (GCC 4.7.2), numpy 1.9.2 and scipy 0.16.0:

```bash
$ python -c 'import scipy; scipy.test()'
Running unit tests for scipy
NumPy version 1.9.2
NumPy is installed in /home/sorge/install-numpy/lib/python3.4/site-packages/numpy
SciPy version 0.16.0
SciPy is installed in /home/sorge/install-numpy/lib/python3.4/site-packages/scipy
Python version 3.4.3 (default, Aug 27 2015, 23:08:30) [GCC 4.7.2]
nose version 1.3.7
...
======================================================================
FAIL: test_kurtosis (test_mstats_basic.TestMoments)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/sorge/install-numpy/lib/python3.4/site-packages/scipy/stats/tests/test_mstats_basic.py", line 414, in test_kurtosis
    stats.kurtosis(self.testcase_2d[2, :]))
  File "/home/sorge/install-numpy/lib/python3.4/site-packages/numpy/testing/utils.py", line 1358, in assert_array_almost_equal_nulp
    raise AssertionError(msg)
AssertionError: X and Y are not equal to 1 ULP (max is 2)

----------------------------------------------------------------------
Ran 18295 tests in 244.049s

FAILED (KNOWNFAIL=98, SKIP=1190, failures=1)
```

Here is some debug output
```python
        import numpy as np
        ax = np.abs(x)
        ay = np.abs(y)
        ref = nulp * np.spacing(np.where(ax > ay, ax, ay))
        if not np.all(np.abs(x-y) <= ref):
            if np.iscomplexobj(x) or np.iscomplexobj(y):
                msg = "X and Y are not equal to %d ULP" % nulp
            else:
                max_nulp = np.max(nulp_diff(x, y))
                msg = "X and Y are not equal to %d ULP (max is %g)" % (nulp, max_nulp)
>           raise AssertionError(msg)
E           AssertionError: X and Y are not equal to 1 ULP (max is 2)

../../../numpy/testing/utils.py:1358: AssertionError
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /home/sorge/install-numpy/lib/python3.4/site-packages/numpy/testing/utils.py(1358)assert_array_almost_equal_nulp()
-> raise AssertionError(msg)
(Pdb) p x, y, ax, ay, nulp, ref, x - y, nulp_diff(x, y)
(-1.4724705238455413, -1.472470523845541, 1.4724705238455413, 1.4724705238455409, 1, 2.2204460492503131e-16, -4.4408920985006262e-16, 2.0)
```

```bash
$ python -c 'import scipy; scipy.show_config()'
atlas_blas_threads_info:
  NOT AVAILABLE
mkl_info:
  NOT AVAILABLE
blas_mkl_info:
  NOT AVAILABLE
atlas_3_10_threads_info:
  NOT AVAILABLE
atlas_3_10_blas_info:
  NOT AVAILABLE
atlas_threads_info:
  NOT AVAILABLE
atlas_info:
    library_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/lib']
    define_macros = [('ATLAS_INFO', '"\\"3.8.4\\""')]
    include_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/include']
    language = f77
    libraries = ['lapack', 'f77blas', 'cblas', 'atlas']
blas_opt_info:
    library_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/lib']
    define_macros = [('ATLAS_INFO', '"\\"3.8.4\\""')]
    include_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/include']
    language = c
    libraries = ['f77blas', 'cblas', 'atlas']
atlas_blas_info:
    library_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/lib']
    define_macros = [('ATLAS_INFO', '"\\"3.8.4\\""')]
    include_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/include']
    language = c
    libraries = ['f77blas', 'cblas', 'atlas']
atlas_3_10_info:
  NOT AVAILABLE
atlas_3_10_blas_threads_info:
  NOT AVAILABLE
lapack_opt_info:
    library_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/lib']
    define_macros = [('ATLAS_INFO', '"\\"3.8.4\\""')]
    include_dirs = ['/usr/nld/atlas-3.8.4-skadi-seq/include']
    language = f77
    libraries = ['lapack', 'f77blas', 'cblas', 'atlas']
openblas_lapack_info:
  NOT AVAILABLE
openblas_info:
  NOT AVAILABLE
lapack_mkl_info:
  NOT AVAILABLE
```

ATLAS has been compiled with GCC 4.3.4

Some system info:
```python
{'architecture': ('64bit', 'ELF'),
 'libc_ver': ('glibc', '2.9'),
 'linux_distribution': ('SUSE Linux Enterprise Server ', '11', 'x86_64'),
 'machine': 'x86_64',
 'packages': {'nose': '1.3.7',
              'numpy': '1.9.2',
              'pip': '7.1.2',
              'pkginfo': '1.2.1',
              'py': '1.4.30',
              'pytest': '2.7.2',
              'readline': '6.2.4.1',
              'scipy': '0.16.0',
              'setuptools': '18.2',
              'six': '1.9.0'},
 'platform_info': 'Linux-2.6.32.59-0.7-default-x86_64-with-SuSE-11-x86_64',
 'platform_release': '2.6.32.59-0.7-default',
 'prefix': '/home/sorge/install-numpy',
 'processor': 'x86_64',
 'python_branch': '',
 'python_build': ('default', 'Aug 27 2015 23:08:30'),
 'python_compiler': 'GCC 4.7.2',
 'python_executable': '/home/sorge/install-numpy/bin/python',
 'python_implementation': 'CPython',
 'python_revision': '',
 'python_version': '3.4.3',
 'python_version_info': sys.version_info(major=3, minor=4, micro=3, releaselevel='final', serial=0),
 'system': 'Linux',
 'system_version': '#1 SMP 2012-07-13 15:50:56 +0200'}
```

--- a/scipy/stats/tests/test_mstats_basic.py
+++ b/scipy/stats/tests/test_mstats_basic.py
@@ -411,7 +411,8 @@ class TestMoments(TestCase):
 
         # Check consistency between stats and mstats implementations
         assert_array_almost_equal_nulp(mstats.kurtosis(self.testcase_2d[2, :]),
-                                       stats.kurtosis(self.testcase_2d[2, :]))
+                                       stats.kurtosis(self.testcase_2d[2, :]),
+                                       nulp=4)
 
     def test_mode(self):
         a1 = [0,0,0,1,1,1,2,3,3,3,3,4,5,6,7]

